.changeset/small-years-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/lib': minor
+---
+
+Adds `Backspace` and `Delete` to `keyMap`
```

Comment:
Looks like we're missing a changeset for Combobox?

Reply:
There's actually no functional changes to Combobox in this PR. Since it's behind another change to Combobox (#1266), I figured this one didn't need a changeset

Reply:
Decided to add a changeset to address some changed tests